cmd/snap-update-ns/change.go

```diff
@@ -116,9 +111,6 @@ func (c *Change) createPath(path string, pokeHoles bool, sec *Secure) ([]*Change
 			// performed the hole poking and thus additional changes must be nil.
 			_, err = c.createPath(path, false, sec)
 		}
-	} else if err != nil {
```

Comment:
That's probably ok and means we will not decorate the error with extra message and just return it as is; is this intended?

Reply:
Yes, as a part of this change the errors produced by things called by createPath internally now return appropriate error messages by themselves.